### PR TITLE
translate testing.md.

### PR DESCRIPTION
**Pull request by yangrisheng**
Just translating ‘testing.md’

### CODE REVIEW

File: docs/zh_cn/testing.md [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    ...
+    val_dataloader=dict(samples_per_gpu=16),
+    test_dataloader=dict(samples_per_gpu=16),
+    ...
+)
+```
+
+将使用 16 张图像作为一个批大小测试模型。
+
+:::{警告}
```
Comment (gaotongxiao) [inline docs/zh_cn/testing.md:106]:
```suggestion
:::{warning}
```

File: docs/zh_cn/testing.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+./tools/dist_test.sh configs/textdet/dbnet/dbnet_r18_fpnc_1200e_icdar2015.py dbnet_r18_fpnc_sbn_1200e_icdar2015_20210329-ba3ab597.pth --eval hmean-iou
+```
+
+下面是脚本的完整用法:
+
+```shell
+python tools/test.py ${CONFIG_FILE} ${CHECKPOINT_FILE} [ARGS]
+```
+
+:::{注}
```
Comment (gaotongxiao) [inline docs/zh_cn/testing.md:19]:
```suggestion
:::{note}
```

**Review by gaotongxiao — COMMENTED** [review]
Hi, thanks for the translation! The special notation is not required to be translated.

**Review by yangrisheng — COMMENTED** [review]
commit change

**Review by yangrisheng — COMMENTED** [review]
change the link of 'model zoo'

File: docs/zh_cn/testing.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
+# 测试
+
+此文档介绍在数据集上测试预训练模型的方法。
+
+## 使用单GPU进行测试
```
Comment (gaotongxiao) [inline docs/zh_cn/testing.md:5]:
```suggestion
## 使用单 GPU 进行测试
```

**Review by gaotongxiao — APPROVED** [review]

**Pull request merged**